Realm/RLMSyncConfiguration.mm

```diff
@@ -129,6 +129,21 @@ - (void)setStopPolicy:(RLMSyncStopPolicy)stopPolicy {
     _config->stop_policy = translateStopPolicy(stopPolicy);
 }
 
+- (NSString *)urlPrefix {
+    if (auto prefix = _config->url_prefix) {
```

Comment:
This is a pointless extra copy of the string. `if (_config->url_prefix) { return @(_config->url_prefix.c_str()); }`